src/wifi/AdafruitIO_AIRLIFT.h

```diff
@@ -158,6 +158,10 @@ class AdafruitIO_AIRLIFT : public AdafruitIO {
     /**************************************************************************/
     void _connect()
     {
+      if(strlen(_ssid) != 0)
```

Comment:
Could you please move your SSID work into a separate branch and open a separate PR for it?

It helps narrow the scope if any issues crop up during tests or releases.

Reply:
Changes made and SSID commit reverted